dashboard/lib/pd/foorm/workshop_summarizer.rb

```diff
@@ -158,5 +158,16 @@ def self.add_question_to_summary(summary_at_question, answer, answers, question_
       end
       summary_at_question
     end
+
+    # Sort summaries so that survey data is ordered in the order surveys
+    # should be taken in. i.e. Pre-Workshop, Day 1, Day 2,.. Post-Workshop
+    def self.sort_summary(workshop_summary)
```

Comment:
I'm not sure how efficient this is...reading says sort_by is not great, but there are probably a maximum of 12 values to sort here in the worst case so it's probably a non-issue. Any suggestions for improvement are appreciated.